readme.md

```diff
@@ -77,13 +77,32 @@ This document contains some additional [technical details](https://github.com/Az
 
-- Use `dotnet pack /p:OfficialBuildId=yyyyMMdd.100` to package up a version of the generator
+- Run `dotnet pack /p:OfficialBuildId=yyyyMMdd.100` at the root directory of this repository to package up a version of the generator
   - Since there can be real versions with the same date starting at `.100` can ensure no conflicts with official builds.
   - This will place a nupkg file at this location `[RepoRoot]\artifacts\packages\Debug`
 - Edit the nuget configuration file [here](https://github.com/Azure/azure-sdk-for-net/blob/main/NuGet.Config) to have a new source `<add key="DEBUG" value="[RepoRoot]\artifacts\packages\Debug" />`
   - This is not intended to be checked in but is a temporary change so `dotnet restore` can find the package
 - Update the [Packages.Data.props](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Packages.Data.props#L156) file to have Microsoft.Azure.AutoRest.Csharp use the new local version you created in step 1
   - This is not intended to be checked in but is a temporary change so `dotnet restore` can find the package
+- If your generator PR needs to apply to cadl projects, or your generator PR changes anything in our emitter (the `src\CADL.Extension\Emitter.Csharp` project), you will need to
+  1. Update the `version` in `src\CADL.Extension\Emitter.Csharp\package.json`
+  2. Run `npm pack` command in `src\CADL.Extension\Emitter.Csharp` directory. This will produce a pack like `src\CADL.Extension\Emitter.Csharp\azure-tools-cadl-csharp-x.y.z.tgz` where `x.y.z` is the version number you set in step 1.
+  3. Change the emitter version used in our `azure-sdk-for-net` repo in this file `eng\emitter-package.json` like this:
+  ```diff
+  {
+    "main": "dist/src/index.js",
+    "dependencies": {
+  -   "@azure-tools/cadl-csharp": "0.1.11-beta.20230212.4"
+  +   "@azure-tools/cadl-csharp": "/absolute/path/to/src/CADL.Extension/Emitter.Csharp/azure-tools-cadl-csharp-x.y.z.tgz"
+    }
+  }
+  ```
+  4. Locate the `dll` file generated for the generator, usually it could be found `autorest.csharp` repository in this directory: `artifacts\bin\AutoRest.CSharp\Debug\net6.0\AutoRest.CSharp.dll`
+  5. Go to [this file](https://github.com/Azure/azure-sdk-for-net/blob/285cbdc343f01774b2b33e758662222bc8a3a33b/eng/common/scripts/Cadl-Project-Generate.ps1#L80) in your local `azure-sdk-for-net` repository, and change it like this by appending a new option to the emitter invocation:
+  ```diff
+  -$cadlCompileCommand = "npx cadl compile $mainCadlFile --emit $emitterName$emitterAdditionalOptions"
```

Comment:
Lets file an issue to make this something we can potentially pass into our dotnet build target and have it get forwarded down

Reply:
I opened this issue to track: https://github.com/Azure/azure-sdk-for-net/issues/34357